dev-client/src/theme.ts

```diff
@@ -1,37 +1,94 @@
 import {extendTheme} from 'native-base';
 
-const newColorTheme = {
-  primary: {
-    main: '#276749',
-    contrast: '#FFFFFF',
-    lightest: '#9AE6B4',
-    // TODO: This is used for the colorScheme value for Radio
-    // We should figure out how the color scheme stuff works and see if we can
-    // map our current variables to 100, 200 values etc.
-    600: '#276749',
+export const theme = extendTheme({
```

Comment:
note: i had quite a bit of trouble figuring out the right format for all these theme fields. i'm not sure i understand well enough to write a helpful comment yet, but hopefully i will be able to at some point. lmk if you have any questions while modifying/adding to this code!